spec/operators/windowCount-spec.js

```diff
@@ -3,31 +3,103 @@ var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
 
 describe('Observable.prototype.windowCount', function () {
-  function mergeMapfunction(x) {
-    return x.toArray();
-  }
+  it('should emit windows with count 2 and skip 1', function () {
+    var source = hot('^-a--b--c--d--|');
+    var subs =       '^             !';
+    var expected =   'u-v--x--y--z--|';
+    var u = cold(    '--a--(b|)      ');
+    var v = cold(      '---b--(c|)   ');
+    var x = cold(         '---c--(d|)');
+    var y = cold(            '---d--|');
+    var z = cold(               '---|');
```

Comment:
Internal behaviorwise I think that makes sense, but also I think it brings sort of confusions who uses those operators such as pre-RxJS4 issue asked about this behavior https://github.com/Reactive-Extensions/RxJS/issues/148, notable thing is some of operators does not behaves in same way due to it's internal implementation behavior. (in case of `buffer*` it makes sense since buffered collection will emit when selector's closing though). Maybe need additional documentation for this topics in further.